recipes/grakel/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "grakel" %}
+{% set version = "0.1.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/grakel-{{ version }}.zip
+  sha256: 44cbfbeadc3d7cf8de22fdb59081b2fc36b9a2cbdeedd848ac6ea85208009a31
+
+build:
+  number: 0
+  skip: true   # [py>=40]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - numpy
+    - python
+    - cython
+  run:
+    - cython >=0.27.3
```

Comment:
`cython` is only needed for build-time, no need to have it under `run`.

Reply:
I removed cython from the run section

Reply:
I had to add cython again otherwise pip check fails.

Reply:
Ugh. It's in the upstream package metadata. Ok, fine.